tests/jerry/es.next/class.js

```diff
@@ -49,6 +49,11 @@ must_throw("class class {}");
 must_throw("class A { constructor() {} this.a = 5 }");
 must_throw("class A { constructor() {} constructor() {} }");
 must_throw("class A { static prototype() {} }");
+must_throw("class A { static get prototype() {} }");
+must_throw("class A { static set prototype() {} }");
```

Comment:
I would add some testcases for the string variant as well:
`must_throw("class A { static get 'prototype'() {} }");`